Sources/WalletConnectSign/Engine/Common/ApproveEngine.swift

```diff
@@ -82,7 +81,15 @@ final class ApproveEngine {
         let proposeResponse = SessionType.ProposeResponse(relay: relay, responderPublicKey: selfPublicKey.hexRepresentation)
         let response = JSONRPCResponse<AnyCodable>(id: payload.wcRequest.id, result: AnyCodable(proposeResponse))
 
+        guard var pairing = pairingStore.getPairing(forTopic: payload.topic) else {
```

Comment:
do we care about pairing is active / inactive ?

Reply:
I think on wallet pairing is always active

Reply:
exaclty